main/src/cgeo/geocaching/filters/core/StatusGeocacheFilter.java

```diff
@@ -75,6 +77,7 @@
     private Boolean statusHasOfflineLog = null;
     private Boolean statusHasOfflineFoundLog = null;
     private Boolean statusSolvedMystery = null;
+    private Boolean statusHasUserDefineWaypoint = null;//Volthai7us
```

Comment:
Please remove the comments with your GitHub name (//Voltahai7us) from the PR. We don't do this (would make code pretty messy over time :-)). `git blame` can be used anytime to find out who did what if need be.

Reply:
ok now, thank you